sale_order_revision/models/sale_order.py

```diff
@@ -35,7 +35,7 @@ def _prepare_revision_data(self, new_revision):
     def action_view_revisions(self):
         self.ensure_one()
         action = self.env.ref("sale.action_orders")
-        result = action.read()[0]
+        result = action.sudo().read()[0]
```

Comment:
@ps-tubtim Nope, you should use _for_xml_id() instead (Look after into code for examples)

Reply:
@rousseldenis Thank you for your suggestion. I already updated it.